src/plots/cartesian/dragbox.js

```diff
@@ -168,13 +171,31 @@ module.exports = function dragBox(gd, plotinfo, x, y, w, h, ns, ew) {
             else if(isSelectOrLasso(dragModeNow)) {
                 dragOptions.xaxes = xa;
                 dragOptions.yaxes = ya;
+
+                // take over selection polygons from prev mode, if any
+                if((e.shiftKey || e.altKey) && (plotinfo.selection && plotinfo.selection.polygons) && !dragOptions.polygons) {
```

Comment:
Would it be possible to move this block into `prepSelect` so that all subplot types that support selections can allow multi-select? Right now, only cartesian and gl2d plot types allow it.

Reply:
Yep